packages/payments/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@thirdweb-dev/payments",
+  "version": "1.0.0",
```

Comment:
since we're bumping `major` this should start out as `0.x.x` so that the release will bump to `1.0.0`